docs/manuals/core/versioning/configuration.md

```diff
@@ -76,7 +76,7 @@ configure the version management system:
   If the version provided already exists as an *experiment* version, it is converted
   to a production version.
 
-- **--taipy-force**: With the `--taipy-force` argument, Taipy overrides a version even if
+- **--force**: With the `--force` argument, Taipy overrides a version even if
```

Comment:
Same here. Choose a font for command line options, and keep it.
In my opinion, regular italics is fine.